.github/workflows/test.sh

```diff
@@ -5,3 +5,32 @@ set -e -u -o pipefail
 VERSION=${1:-stable}
 
 rustup run ${VERSION} cargo test --verbose
+
+# CLI tests
+rustup run ${VERSION} cargo build --verbose
+
+# TODO add additional targets to tests when ready
```

Comment:
Right now we only have a debug target.  I am hesitant to just do all subdirs in the targets directory since that might be where our doc stuff ends up.

Reply:
Okay, this is fine for now.  I think that independent of this, we should start doing release builds in the CI and test that as well.